composer.json

```diff
@@ -3,9 +3,9 @@
     "homepage": "http://github.com/c9s/ClassTemplate",
     "description": "Class template Utilities",
     "require": {
-        "php": ">=5.3.0",
-        "corneltek/codegen": "^2",
-        "twig/twig": "^1"
+        "php": ">=7.2",
```

Comment:
This might break the dependent packages

Reply:
Alright, will try to use an old php version, is 7.1 ok? or should it be 5.x ?